TCMP/BasicNFC-CommandFamily/responses/TagWrittenResponse.swift

```diff
@@ -45,7 +45,7 @@ import Foundation
 
     @objc public override init(){}
     
-    public init(tagCode : [UInt8], tagType: TagTypes){
+    @objc public init(tagCode : [UInt8], tagType: TagTypes){
```

Comment:
I will incorporate this into the next release

Reply:
Thanks!